### PR TITLE
add card

### PR DESCRIPTION
**Pull request by Justintnwj**
Please Review My Friends

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,57 +2,72 @@
...
+    </button>
+    <button class="carousel-control-next" type="button" data-bs-target="#carouselExample" data-bs-slide="next">
+      <span class="carousel-control-next-icon" aria-hidden="true"></span>
+      <span class="visually-hidden">Next</span>
+    </button>
+  </div>
+
+  <!-- Card Section by Just -->
+  <div class="card" style="width: 18rem;">
+    <img src="..." class="card-img-top" alt="...">
```
Comment (joviantomd) [inline index.html]: Add image

**Review by joviantomd — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Justintnwj) [reply index.html]: Done ya kakak

File: index.html [hunk 2]
```diff
@@ -2,57 +2,72 @@
...
+    <div class="card-body">
+      <h5 class="card-title">Card titl exca</h5>
```
Comment (IkaPurnama) [inline index.html]: Change the title to  Excavators

**Review by IkaPurnama — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (Justintnwj) [reply index.html]: oke kakak

**Review by joviantomd — APPROVED** [review]

**Review by IkaPurnama — APPROVED** [review]

**Pull request merged**